keras/ops/nn.py

```diff
@@ -1633,14 +1651,14 @@ def multi_hot(inputs, num_classes=None, axis=-1, dtype=None, **kwargs):
         raise ValueError("Argument `num_classes` must be specified.")
 
     if any_symbolic_tensors((inputs,)):
-        return MultiHot(num_classes, axis, dtype).symbolic_call(inputs)
+        return MultiHot(num_classes, axis, dtype, sparse).symbolic_call(inputs)
 
-    return backend.nn.multi_hot(inputs, num_classes, axis, dtype)
+    return backend.nn.multi_hot(inputs, num_classes, axis, dtype, sparse)
 
 
 class Moments(Operation):
-    def __init__(self, axes, keepdims=False, synchronized=False, name=None):
-        super().__init__(name)
+    def __init__(self, axes, keepdims=False, synchronized=False):
+        super().__init__()
```

Comment:
Shouldn't we have `**kwargs` in these op classes (for `name`)?

Reply:
I just made it consistent with the other ops. Almost all ops have no `name` (the op name is inferred from the class name) and no `**kwargs`. The few I fixed were the outliers.
AFAICT, `name` is never used.